docs/pages/contribution/testing/index.md

```diff
@@ -4,9 +4,11 @@ Testing is critical to the Lodestar project and there are many types of tests th
 
 There are few flags you can set through env variables to override behavior of testing and it's output.
 
-| ENV variable      | Effect | Impact                                                                                                      |
-| ----------------- | ------ | ----------------------------------------------------------------------------------------------------------- |
-| TEST_COMPACT_DIFF | All    | Will strip down the object difference rendered during test failures. Very useful for large object matching. |
+| ENV variable        | Effect | Impact                                                                                                                    |
+| ------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------- |
+| TEST_COMPACT_DIFF   | All    | Will strip down the object difference rendered during test failures. Very useful for large object matching.               |
+| TEST_QUIET_CONSOLE  | All    | Will strip down console output. Reduce console flickering.                                                                |
```

Comment:
why do you mean by "strip down" here? doesn't this disable console logging if set?

Reply:
There's still the `vitest` output, this disables `lodestar` console output. Open to description clarifications.